imap_processing/glows/l0/decom_glows.py

```diff
@@ -50,6 +50,11 @@ def decom_packets(
     histdata = []
     dedata = []
 
+    filename = packet_file_path.split("/")[-1]
```

Comment:
This will fail our Windows friends. Use pathlib?
```suggestion
    filename = Path(packet_file_path).name
```